src/relax/op/ccl/ccl.cc

```diff
@@ -50,6 +50,44 @@ TVM_REGISTER_OP("relax.ccl.allreduce")
     .set_attr<FRelaxInferLayout>("FRelaxInferLayout", InferLayoutUnaryEwise)
     .set_attr<Bool>("FPurity", Bool(true));
 
+/* relax.ccl.allgather */
+TVM_REGISTER_NODE_TYPE(AllGatherAttrs);
```

Comment:
As `AllGatherAttrs` is no longer used, this `TVM_REGISTER_NODE_TYPE` declaration can be removed.